ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -486,3 +486,25 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None, aweights=N
     return ivy.cov(
         m, y, rowVar=rowvar, bias=bias, ddof=ddof, fweights=fweights, aweights=aweights
     )
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes(
+    {"0.4.13 and below": ("complex64", "complex128", "bfloat16", "bool", "float16")},
+    "jax",
+)
+def quantile(
+    a,
+    q,
+    /,
+    *,
+    axis=None,
+    out=None,
+    overwrite_input=False,
+    method="linear",
+    keepdims=False,
+    interpolation=None,
+):
+    return ivy.quantile(
```

Comment:
We can handle this by adding a simple condition here that if the mode is `nearest`, we can pass in `nearest_jax` to the ivy function
```
if mode=="nearest":
    ivy.quantile(    a, q, axis=axis, keepdims=keepdims, interpolation='nearest_jax', out=out
    )
```

Thanks! :slightly_smiling_face: